cmd/create/cluster/cmd.go

```diff
@@ -644,6 +649,14 @@ func run(cmd *cobra.Command, _ []string) {
 		isIAM = !isSTS
 	}
 
+	isSTS = isSTS || awsCreator.IsSTS
+
+	if r.Reporter.IsTerminal() {
+		r.Reporter.Warnf("In a future release STS will be the default mode.")
+		r.Reporter.Warnf("--sts flag won't be necessary if you wish to use STS.")
```

Comment:
I've noticed that previously we printed the flag messages conditionally.
Should we preserve it with `if` statements?

Reply:
I thought of removing it to bring more attention to the user. I'd users are using --sts, so they need to be alerted about the other option as well if they ever want to use it. Since the other option is a new param they would probably never see the message unless someone decides to check --help